db/osqlcomm.c

```diff
@@ -4814,9 +4814,11 @@ int osql_send_commit_by_uuid(char *tohost, uuid_t uuid, int nops,
             sbuf2flush(logsb);
         }
 
+#if 0
```

Comment:
Without using #if 0 here, this output ends up as stray in one of the test cases (random_osql_replay).